lib/localeapp/cli/install.rb

```diff
@@ -8,29 +8,31 @@ def initialize(args = {})
         @config_type = :default
       end
 
-      def execute(key = nil)
-        installer("#{config_type.to_s.capitalize}Installer").execute(key)
+      def execute(key = nil, options = {})
+        installer("#{config_type.to_s.capitalize}Installer").execute(key, options)
       end
 
       def installer(installer_class)
         self.class.const_get(installer_class).new(@output)
       end
 
       class DefaultInstaller
-        attr_accessor :key, :project_data, :config_file_path, :data_directory
+        attr_accessor :key, :project_data, :config_file_path, :data_directory, :apikey_to_dotenv
 
         def initialize(output)
           @output = output
         end
 
-        def execute(key = nil)
+        def execute(key = nil, options = {})
           self.key = key
+          apikey_to_dotenv = options[:apikey_to_dotenv]
           print_header
           if validate_key
             check_default_locale
             set_config_paths
             @output.puts "Writing configuration file to #{config_file_path}"
             write_config_file
+            write_apikey_to_dotenv if apikey_to_dotenv
```

Comment:
It is further used in `config_api_key_str`. If you find it preferable to get rid of `apikey_to_dotenv`, I can remove it and pass the information around in method arguments where necessary.